examples/server/webui/src/main.js

```diff
@@ -3,6 +3,14 @@ import { createApp, defineComponent, shallowRef, computed, h } from 'vue/dist/vu
 import MarkdownIt from 'markdown-it';
 import TextLineStream from 'textlinestream';
 
+// code highlighting
+import hljs from 'highlight.js';
+import HighlightJS from 'markdown-it-highlightjs' // used integrate highlight.js into markdown-it
+import 'highlight.js/styles/github-dark.min.css';
+// math formula rendering
+import 'katex/dist/katex.min.css'
```

Comment:
```suggestion
import 'katex/dist/katex.min.css';
```

Please respect existing style